lib/flags.mli

```diff
@@ -92,6 +92,10 @@ val is_universe_polymorphism : unit -> bool
 val make_polymorphic_inductive_cumulativity : bool -> unit
 val is_polymorphic_inductive_cumulativity : unit -> bool
 
+(** Global uniform inductive parameters flag *)
+val make_uniform_inductive_parameters : bool -> unit
+val is_uniform_inductive_parameters : unit -> bool
```

Comment:
@ejgallego Do you want to comment on the appropriate location for defining a new flag / option? I seem to recall that we prefer to define options at the point where they are used nowadays.

Reply:
Well indeed this is documented at the top of `Flags.mli`.

Please move the flags to the proper module. Also, don't pollute the namespace with `make/is` fucntions.

Reply:
Sorry about that, I was blindly mimicking the cumulativity flag above, and didn't read `flags.mli`. I moved the option to `vernacentries.ml`, which is where it is used, but if it would be better in `ComInductive` let me know.